packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -0,0 +1,130 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import tokens from "@paprika/tokens";
+
+const integize = token => Number.parseInt(token, 10);
+
+const sizeMap = {
+  [ShirtSizes.SMALL]: 2,
+  [ShirtSizes.MEDIUM]: 2.5,
+  [ShirtSizes.LARGE]: 3,
+};
+
+const getSpacing = size => stylers.spacer(sizeMap[size]);
+const getSpacingInt = size => integize(getSpacing(size));
+
+const size = ({ size }) => getSpacing(size);
+const fontSizeInt = size => (getSpacingInt(size) / integize(tokens.space) - 3) * 2;
+const fontSizeValue = ({ size }) => `${stylers.fontSizeValue(fontSizeInt(size))}px`;
+
+const topOffset = size => (getSpacingInt(size) - integize(stylers.fontSizeValue()) * stylers.lineHeightValue(-1)) / 2;
+const isCheckboxBigger = size => topOffset(size) > 0;
+const labelPadding = ({ hasChildren, size }) => {
+  const top = isCheckboxBigger(size) ? `${topOffset(size)}px` : 0;
+  const left = hasChildren ? `${getSpacingInt(size) + integize(tokens.space)}px` : getSpacing(size);
+  return `${top} 0 ${top} ${left}`;
+};
+
+const checkerTop = size => (isCheckboxBigger ? "-1px" : `${Math.abs(topOffset(size) / 2)}px`);
+const checkBoxIconLeft = ({ size }) => `${getSpacingInt(size) / 2}px`;
```

Comment:
This is a bit of a rat's nest of complexity. Could it be simplified so guys like me can understand it? (map it, dont calculate it)